public/javascript/posts/BookAppointment.js

```diff
@@ -19,6 +19,24 @@ var app = angular.module('BookAppointmentApp');
     ];
 
     $scope.viewRenderWeekAgenda = function(view, element){
+      if(view.type == 'month'){
+        $('.fc-day-number').css('cursor', 'pointer');
+        $('.fc-day').css('cursor', 'pointer');
+        $('.fc-week').css('cursor', 'pointer');
+        $('.fc-week').each(function(){
+          $(this).mouseover(function(){
+            $(this).tooltip();
+            $(this).attr('title', 'Click to see week view');
+          }).mouseleave(function(){

```

Comment:
Check the function `eventRenderContent` where tooltip is attached and `dayEvent` where it is closed